migrations/20150926071544-team-status-migration.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+module.exports = {
+	up: function (migration, DataTypes) {
+		/*
+			Add altering commands here.
+			Return a promise to correctly handle asynchronicity.
+		*/
+
+		migration.createTable('team_statuses', {
+			id: {
+				type: DataTypes.INTEGER.UNSIGNED,
+				primaryKey: true,
+				autoIncrement: true,
+				allowNull: false
+			},
+			team: {
+				type: DataTypes.CHAR,
+				length: 3,
+				allowNull: false
+			},
+			status: {
+				type: DataTypes.CHAR,
+				length: 15,
+				allowNull: false
+			},
+			bought_by: {
```

Comment:
"triggered_by" would be a more generic term in case we want some kind of effect that wasn't "bought". Just a thought.